snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: sidekick
+version: "0.0.6"
```

Comment:
We need to change this also during the build process. This can be a bash script within the `/scripts` directory

Reply:
Isso seria um problema, pois para o snap deve se respeitar essa configuração de pasta

Esse valor realmente terá que ser mudado manualmente, mas isso necessariamente não seria um problema pois a builder do snap não seria feita pelo github actions e sim pela loja de aplicativos [snapcraft](https://snapcraft.io/)

Reply:
Não podemos configurar a CI, para set a versão pra a do tag? Estamos fazendo o mesmo pro windows aqui https://github.com/leoafarias/sidekick/blob/main/.github/workflows/release.yml#L168

Reply:
Para o app appImage sim, mas para o snap é um pouco mais complicado

Pois snaps fica observando o repositório em si e acada commit ele lança uma nova versão no canal edge e por la você decide se a versão vai pro canal beta ou stable.

Eu poderia fazer como o [nodejs](https://github.com/nodejs/snap) fez, mais ainda é um pouco complexo pra mim 

Eles basicamente fazem a alterações na ci e mandam para o repositório principal por meio de commit https://github.com/nodejs/snap/blob/master/snapcraft.yaml.sh